docs/reference/contributing/target/QuadSPI.md

```diff
@@ -1,25 +1,54 @@
 <h2 id="quadspi-port">QuadSPI</h2>
 
-[Include a brief description here.]
+Implementing QSPI enables Mbed OS to communicate with external memories much faster than via SPI.
```

Comment:
@0xc0170 One outstanding item: Please expand this paragraph. What does this API do, and what are its use cases?

Reply:
Added in cefda8b